AppCreationScripts/sample.json

```diff
@@ -12,19 +13,25 @@
   "AADApps": [
     {
       "Id": "service",
-      "Name": "TodoListService",
+      "Name": "TodoListService-Core-Cert",
       "IsPublicClient": false,
```

Comment:
IsPublicClient: not needed

Reply:
@jmprieur could you please update it here first? https://github.com/Azure-Samples/active-directory-dotnet-daemon/blob/master/AppCreationScripts/sample.json
This `sample.json` is still new to me and used yours as the source of truth. So, do you mind updating it so I can have a recent reference?